external/vcm/vcm/convenience.py

```diff
@@ -146,3 +146,27 @@ def shift_timestamp(time: str, seconds: Union[int, float]) -> str:
 def get_root():
     """Returns the absolute path to the root directory for any machine"""
     return str(TOP_LEVEL_DIR)
+
+
+def gsrm_name_from_resolution_string(res: str):
+    """Returns the GSRM name based on the resolution"""
+    if res.startswith("ne"):
+        gsrm = "scream"
+    elif res.startswith("c"):
+        gsrm = "fv3"
+    else:
+        raise ValueError(
+            f"This resolution {res} can not be mapped to either scream or fv3."
+        )
+    return gsrm
+
+
+def horizontal_dims_from_resolution_string(res: str):
```

Comment:
I would delete this function, and instead just have code in `fv3net/diagnostics` to return the appropriate dim names based on the output from `gsrm_name_from_resolution_string`. Right now, we've defined the FV3 horizontal dims in two places (here, and as `HORIZONTAL_DIMS_FV3` in `fv3net/diagnostics/_shared/constants.py`, which is not ideal.